### PR TITLE
reorganizing to make sense, add slack guides, etc

### PR DESCRIPTION
**Pull request by parispittman**
Added slack guidelines, communications folder
Reorganized the communication and event docs that were in community/community
Added an events folder
Made edits to communication.md

### CODE REVIEW

**Comment by parispittman** [comment]
cc/ @jdumars

File: communication.md [hunk 1]
```diff
@@ -19,19 +19,16 @@ and meetings devoted to Kubernetes.
 
 ## Social Media
 
-* [Twitter]
```
[the remark this answers is not in the document]
Comment (parispittman) [reply communication.md]: Duh - And now I know another markdown tip :) - thanks for catching that. Thought those were dead links for some reason.

File: communication.md [hunk 2]
```diff
@@ -20,18 +20,15 @@ and meetings devoted to Kubernetes.
 ## Social Media
 
 * [Twitter]
-* [Google+]
-* [blog]
-* Pose questions and help answer them on [Slack][slack.k8s.io] or [Stack Overflow].
+* [Blog]
```
Comment (jdumars) [inline communication.md]: [Blog](https://blog.k8s.io/)

File: communication.md [hunk 3]
```diff
@@ -20,18 +20,15 @@ and meetings devoted to Kubernetes.
...
+* Pose questions and help answer them on [Stack Overflow].
```
Comment (jdumars) [inline communication.md]: ```[Stack Overflow](https://stackoverflow.com/tags/kubernetes/info)```

File: communication.md [hunk 4]
```diff
@@ -20,18 +20,15 @@ and meetings devoted to Kubernetes.
...
+* [Slack] - sign up
```
Comment (jdumars) [inline communication.md]: ```[Slack](http://slack.k8s.io/)```

File: communication.md [hunk 5]
```diff
@@ -20,18 +20,15 @@ and meetings devoted to Kubernetes.
...
-Most real time discussion happens at [kubernetes.slack.com];
-you can sign up at [slack.k8s.io].
- 
+Real time discussion at kubernetes.slack.io:   
 Discussions on most channels are archived at [kubernetes.slackarchive.io].
 Start archiving by inviting the _slackarchive_ bot to a
 channel via `/invite @slackarchive`.
-To add new channels, contact one of the admins
-(briangrant, goltermann, jbeda, sarahnovotny and thockin).
+To add new channels, contact one of the admins in the #slack-admins channel. Our guidelines are [here](/communication/slack-guidelines.md).
```
Comment (jdumars) [inline communication.md]: I'm not sure where this link is supposed to go.

File: communication/slack-guidelines.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+What if you have a problem with an admin?
+Send a DM to another listed Admin and describe the situation OR
+If it’s a code of conduct issue, please send an email to steering-private@kubernetes.io and describe the situation
+
+## BOTS, TOKENS, WEBHOOKS, OH MY
+
+Bots, tokens, and webhooks are reviewed on a case-by-case basis with most requests being rejected due to security, privacy, and usability concerns.. Bots and the like tend to make a lot of noise in channels. Our Slack instance has over 30,000 people and we want everyone to have a great experience. Please join #Slack-admins and have a discussion about your request before requesting the access. GitHub workflow alerts into certain channels and requests from CNCF are typically OK.
+
+## ADMIN MODERATION
+Be mindful of how you handle communication during stressful interactions.  Administrators act as direct representatives of the community, and need to maintain a very high level of professionalism at all times. If you feel too involved in the situation to maintain impartiality or professionalism, that’s a great time to enlist the help of another admin.
```
[the remark this answers is not in the document]
Comment (jdumars) [reply communication/slack-guidelines.md]: +1 but not in this PR -- I'd like to have a longer discussion as a team about that, and not hold up this reorg until then.

File: communication.md [hunk 7]
```diff
@@ -52,11 +49,11 @@ Users trade notes on the Google group
 
 ## Office Hours
 
-Office hours are held once a month. Please refer to [this document](community/office-hours.md) to learn more. 
+Office hours are held once a month. Please refer to [this document](community/office-hours.md) to learn more.
```
Comment (cblecker) [inline communication.md]: This link is broken now

**Pull request merged**